mwdb/web/src/components/Profile/Views/ProfileOauth.js

```diff
@@ -0,0 +1,112 @@
+import React, { useContext, useEffect, useState } from "react";
+
+import { APIContext } from "@mwdb-web/commons/api/context";
+
+import { faPlus } from "@fortawesome/free-solid-svg-icons";
+import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
+
+import { ConfirmationModal, useViewAlert } from "@mwdb-web/commons/ui";
+
+export function ProfileOauth() {
```

Comment:
`OAuth` instead of `Oauth`
```suggestion
export function ProfileOAuth() {
```